lib/presentation/shared_widgets/custom_appbar.dart

```diff
@@ -0,0 +1,33 @@
+import 'package:auto_route/auto_route.dart';
+import 'package:flutter/material.dart';
+
+import '../themes/constants.dart';
+
+class CustomAppBar extends StatelessWidget implements PreferredSizeWidget {
+  final BuildContext _context;
+  final String title;
+  const CustomAppBar(this._context, {Key? key, required this.title})
+      : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    return AppBar(
+      backgroundColor: kSecondaryColor,
+      elevation: 0.0,
+      centerTitle: true,
+      leading: GestureDetector(
+        onTap: () => context.router.pop(),
+        child: const Icon(
+          Icons.arrow_back_ios_new,
```

Comment:
This is different from the "back" icon in Figma, right?
Please use this asset or mark this section with a TODO.